packages/e2e-tests/specs/editor/various/reusable-blocks.test.js

```diff
@@ -165,40 +209,19 @@ describe( 'Reusable blocks', () => {
 	} );
 
 	it( 'can be inserted after refresh', async () => {
```

Comment:
After investigating, I learned that `can be inserted and edited` and `can be inserted after refresh` are different tests.

* `can be inserted and edited` focuses on testing creating and editing in one writing session.
* `can be inserted after refresh` focuses on testing the reusable block is saved correctly to the database and loaded without fail when writing a new post or editing existing ones.